Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -15,6 +15,7 @@ public protocol PledgeAmountViewModelInputs {
   func stepperValueChanged(_ value: Double)
   func textFieldDidEndEditing(_ value: String?)
   func textFieldValueChanged(_ value: String?)
+  func updateMaximumPledgeAmount(with shippingCost: Double)
```

Comment:
The input should be named based on the event that occurred, so in this case I think it's `selectedShippingAmountChanged(to amount: Double)`.